CONTRIBUTING.md

```diff
@@ -45,6 +45,36 @@ To run code linter, use:
 npm run lint
 ```
 
+## Commit Messages
+
+Commit messages should follow the Semantic Commit Messages format.
+
+Example:
+
+```
+fix(Page): fix page.pizza method
+
+This patch fixes page.pizza so that it works with iframes.
+
+Fixes #123, Fixes #234
+
+BREAKING CHANGE: page.pizza now delivers pizza at home by default.
+To deliver to a different location, use "deliver" option:
+  `page.pizza({deliver: 'work'})`.
+```
+1. *label* is one of the following:
```

Comment:
It's not immediately obvious where each of these items maps to in your example. Is there a way to show an abstract example along with the concrete pizze one?

Like:
```
label(namespace): description

body

footer
```

Reply:
done